### PR TITLE
host: add alternate dns server example

### PR DESCRIPTION
**Pull request by m-messiah**
----
- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/host.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Reverse lookup an IP:
 
 `host {{ip_address}}`
+
+- Specify an alternate DNS server to query (8.8.8.8 is google's public DNS):
```
Comment (sbrl) [inline pages/common/host.md]: Perhaps ...`Google's`... would read better?

File: pages/common/host.md [hunk 2]
```diff
@@ -13,3 +13,7 @@
 - Reverse lookup an IP:
 
 `host {{ip_address}}`
+
+- Specify an alternate DNS server to query (8.8.8.8 is Google's public DNS):
```
Comment (agnivade) [inline pages/common/host.md]: I wonder if it is necessary to specify the "(8.8.8.8 is ..)". Even if people do not know what 8.8.8.8, the description is clear that 8.8.8.8 is a specific dns server which is used to query.

Comment (m-messiah) [reply pages/common/host.md]: I have copied this description from dig https://github.com/tldr-pages/tldr/blob/master/pages/common/dig.md

If it does not necessary - maybe remove it from there too?

Comment (agnivade) [reply pages/common/host.md]: Yes, I think so.

Comment (m-messiah) [reply pages/common/host.md]: I created the separate PR https://github.com/tldr-pages/tldr/pull/2447, cause [contributing guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request)

**Pull request merged**